src/sqlfluff/core/default_config.cfg

```diff
@@ -5,7 +5,7 @@ verbose = 0
 nocolor = False
 # Supported dialects https://docs.sqlfluff.com/en/stable/dialects.html
 # Or run 'sqlfluff dialects'
-dialect = ansi
+dialect = None
```

Comment:
This is a key part of the change. The built-in default config no longer specifies a dialect.